### PR TITLE
CTB - GroupPage - List group attributes and delete attribute 

### PR DESCRIPTION
**Pull request by virginieky**
#### Description of what you did:
Display the list of attributes in the GroupPage and enable deletion

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [ ] 💅 Enhancement
- [X] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [ ] Framework
- [X] Plugin

#### Manual testing done on the following databases:

- [X] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: packages/strapi-plugin-content-type-builder/admin/src/containers/App/reducer.js [hunk 1]
```diff
@@ -258,6 +259,20 @@ function appReducer(state = initialState, action) {
...
+      const attributes = state.getIn(pathToAttributes);
+      const attributeName = action.keys.pop();
+      const attributeToDelete = attributes.findIndex(
+        attribute => attribute.get('name') === attributeName
+      );
+
+      return state.removeIn([...pathToAttributes, attributeToDelete]);
+    }
     case DELETE_GROUP_SUCCEEDED:
       console.log({
```
Comment (soupette) [inline packages/strapi-plugin-content-type-builder/admin/src/containers/App/reducer.js]: Can you remove the log?

File: packages/strapi-plugin-content-type-builder/admin/src/containers/ModelPage/index.js [hunk 2]
```diff
@@ -373,7 +373,14 @@ export class ModelPage extends React.Component {
     this.setState({ attrToDelete: null, showWarning: false });
   };
 
-  handleGoBack = () => this.props.history.goBack();
+  handleGoBack = () => {
+    const {
+      location: { pathname },
+    } = this.props;
+    const backPathname = pathname.substr(0, pathname.lastIndexOf('/'));
```
Comment (soupette) [inline packages/strapi-plugin-content-type-builder/admin/src/containers/ModelPage/index.js]: I'm not sure to understand where it should redirect the user? The homepage with the previous selected tab or just the previous view?

File: packages/strapi-plugin-content-type-builder/controllers/Fixtures.js [hunk 3]
```diff
@@ -25,6 +25,7 @@ module.exports = {
                 model: 'file',
                 via: 'related',
                 plugin: 'upload',
+                type: 'media',
```
Comment (soupette) [inline packages/strapi-plugin-content-type-builder/controllers/Fixtures.js]: The media type doesn't exist it's a relation

File: packages/strapi-plugin-content-type-builder/admin/src/containers/ModelPage/index.js (hunk 2 above)
Comment (virginieky) [reply packages/strapi-plugin-content-type-builder/admin/src/containers/ModelPage/index.js]: It should redirect to the homepage with the tab selected at the feature where the user were. 
Ex: If he comes from a group, it redirects to the homepage with the group tab selected, even if he previously navigated in the leftmenu.

**Pull request merged**